pptx/table.py

```diff
@@ -383,6 +384,109 @@ def vertical_anchor(self):
     def vertical_anchor(self, mso_anchor_idx):
         self._tc.anchor = mso_anchor_idx
 
+    @property
+    def borL(self):
+        """
+        Get Left Border instance containing all the formatting options.
+        """
+        return self._tc.tcPr.borL
+
+    @lazyproperty
+    def border_left(self):
+        return BorderFormat(self, 'left')
+
+    def get_or_add_borL(self):
+        return self._tc.tcPr.get_or_add_borL()
+
+    @property
+    def borR(self):
+        """
+        Get Right Border instance containing all the formatting options.
+        """
+        return self._tc.tcPr.borR
+
+    @lazyproperty
+    def border_right(self):
+        return BorderFormat(self, 'right')
+
+    def get_or_add_borR(self):
+        return self._tc.tcPr.get_or_add_borR()
+
+    @property
+    def borT(self):
+        """
+        Get Top Border instance containing all the formatting options.
+        """
+        return self._tc.tcPr.borT
+
+    @lazyproperty
+    def border_top(self):
+        return BorderFormat(self, 'top')
+
+    def get_or_add_borT(self):
+        return self._tc.tcPr.get_or_add_borT()
+
+    @property
+    def borB(self):
+        """
+        Get Bottom Border instance containing all the formatting options.
+        """
+        return self._tc.tcPr.borB
+
+    @lazyproperty
+    def border_bottom(self):
+        return BorderFormat(self, 'bottom')
+
+    def get_or_add_borB(self):
+        return self._tc.tcPr.get_or_add_borB()
+
+
+    @property
+    def borTlBr(self):
+        """
+        Get Diagonal Border instance from Top Left to Bottom Right containing all the formatting options.
+        """
+        return self._tc.tcPr.borB
+
+    @lazyproperty
+    def border_tl_br(self):
+        return BorderFormat(self, 'tl_br')
+
+    def get_or_add_borTlBr(self):
+        return self._tc.tcPr.get_or_add_borTlBr()
+
+
+    @property
+    def borBlTr(self):
+        """
+        Get Diagonal Border instance from Bottom Left to Top Right containing all the formatting options.
+        """
+        return self._tc.tcPr.borBlTr
```

Comment:
same with these function (borBlTr, borTlBr, borB, borT, etc..) necessary in _Cell?

Reply:
You are correct.  I've learned a lot since i first wrote these and it is now greatly simplified!